independent-projects/qute/core/src/main/java/io/quarkus/qute/ImmutableList.java

```diff
@@ -5,15 +5,12 @@
 import java.util.Arrays;
 import java.util.Collection;
 import java.util.Collections;
-import java.util.Comparator;
 import java.util.Iterator;
 import java.util.List;
 import java.util.ListIterator;
 import java.util.NoSuchElementException;
 import java.util.Spliterator;
 import java.util.Spliterators;
-import java.util.function.Predicate;
-import java.util.function.UnaryOperator;
 
 /**
  * Immutable lists.
```

Comment:
Do we still need this class? What about using `List.of`?

Reply:
Well, List.of() does not permit null elements + this class is a bit more optimized for qute 😉...